src/main/java/com/google/devtools/build/lib/bazel/rules/python/python_stub_template.txt

```diff
@@ -275,6 +275,8 @@ def Main():
   python_path_entries = CreatePythonPathEntries(python_imports, module_space)
   python_path_entries += GetRepositoriesImports(module_space, %import_all%)
 
+  # Remove any duplicates to reduce path length
```

Comment:
Nit: It's not the individual paths that are being shortened, but the overall `PYTHONPATH` length.